src/utils/mapUtils.tsx

```diff
@@ -5,14 +5,16 @@ import French from "i18n-iso-countries/langs/fr.json";
 import { Layer, LeafletMouseEvent } from 'leaflet';
 import React from 'react';
 import ReactDOMServer from 'react-dom/server';
-import { Map, MapProps } from 'react-leaflet';
 import { LanguageType, AggregatedRecord, EstimateGradePrevalence, RegionalPrevalenceEstimate } from "../types";
 import { toPascalCase } from './translate/caseChanger';
 import Translate from './translate/translateService';
+import _ from "lodash";
 
-export const getBuckets = (features: GeoJSON.Feature[]) => {
+export const getBuckets = (features: any) => {
+  if (!features)
+    return [0];

```

Comment:
I would add in some curly braces here for this if statement, we are one bad linter away from buckets being broken